python_src/src/lamp_py/aws/s3.py

```diff
@@ -61,7 +61,9 @@ def upload_file(
 
         s3_client = get_s3_client()
 
-        s3_client.upload_file(file_name, bucket, object_name, extra_args)
+        s3_client.upload_file(
+            file_name, bucket, object_name, ExtraArgs=extra_args
+        )
```

Comment:
Added `ExtraArgs` and named parameter per: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/s3-uploading-files.html

Reply:
can we add the extra args to the `upload_log`?

Reply:
Incorporated!